gnome/gtk3-devel/Portfile

```diff
@@ -92,6 +92,9 @@ patchfiles-append   patch-quartz-meson-appleframeworks.diff
 # implicit declaration of function 'sincos'
 patchfiles-append   patch-tests-gdkgears.diff
 
+# https://github.com/mesonbuild/meson/issues/7652
```

Comment:
You're patching gtk but you're referencing a meson bug. If this is a gtk bug, is there a gtk bug report to reference?

Reply:
I think this is a meson bug, since detecting a function where it is not available is just wrong. The fix simply gets around that.
Meson also misdetects some other functions in a similar manner. There is a ticket for those on Trac and on Meson.